core/storage/user/gae_models.py

```diff
@@ -166,7 +187,7 @@ def get_export_policy(cls):
                 base_models.EXPORT_POLICY.EXPORTED,
             'first_contribution_msec':
                 base_models.EXPORT_POLICY.EXPORTED,
-            'pin': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            'pin': base_models.EXPORT_POLICY.EXPORTED
```

Comment:
This is an auth mechanism. We probably shouldn't export this?

Reply:
done